### PR TITLE
Static dependencies

### PR DESCRIPTION
**Pull request by ftdebugger**
Usage:

```js
createContainer({
    dependencies: {
        User: {
             static: {
                  loadStats: true
             }
        }
    }
})
```

### CODE REVIEW

**Comment by jakwuh** [comment]
LGTM

**Comment by next0** [comment]
LGTM

File: index.js [hunk 1]
```diff
@@ -608,6 +616,10 @@ let normalizeDefinitions = (dependencies) => {
                     definition.dependencies = extend({}, parent.dependencies, definition.dependencies);
                     definition.dependencies = omitBy(definition.dependencies, value => value == null);
                 }
+
+                if (parent.static) {
+                    definition.static = extend({}, parent.static, definition.static);
```
Comment (vlyahovich) [inline index.js:621]: Maybe Object.assign and remove lodash deps around?

File: index.js [hunk 2]
```diff
@@ -898,7 +910,13 @@ let createContainer = ({
      * @returns {Promise<object>|object}
      */
     let loadModuleDependencies = (definition, params) => {
-        return loadModules(definition.dependencies, params);
+        return then(loadModules(definition.dependencies, params), dependencies => {
+            if (definition.static) {
+                return extend(dependencies, definition.static);
```
Comment (vlyahovich) [inline index.js:915]: This too

**Comment by vlyahovich** [comment]
Looks good overall, but please consider lodash dep

File: index.js (hunk 1 above)
Comment (next0) [reply index.js:621]: here is no any deps anymore. all functions like `extend`, `omit`, `find` and etc. implemented in a top of file.

**Comment by ftdebugger** [comment]
There are no lodash dependency

**Pull request merged**